packages/core/src/types.ts

```diff
@@ -50,6 +50,7 @@ export interface InitConfig {
   useDidSovPrefixWhereAllowed?: boolean
   connectionImageUrl?: string
   autoUpdateStorageOnStartup?: boolean
+  autoCreateConnectionOnPing?: boolean
```

Comment:
I think we should add this to the connections module config

Reply:
In case we don't use trust ping to create connection, this will apply as well? I guess so, but should have a different naming.